Examples/Filtering/SpatialObjectToImage3.cxx

```diff
@@ -166,7 +166,9 @@ int main( int argc, char *argv[] )
     point = center + radial;
     polygonPoint.SetPositionInObjectSpace( point );
     polygon->GetPoints().push_back( polygonPoint );
+    std::cout << point[0] << ", " << point[1] << std::endl;
```

Comment:
Is this debugging left-over?

Reply:
Looking below in the code, I see that there are other `std::cout` calls. If this one is here to stay, maybe a description of what is printed should be added (see line 175).

Reply:
I will remove this line from the PR and merge the PR. If the `std::cout` was intended, please create a new PR to add it back.